data/json/items/gun/9mm.json

```
@@ -457,11 +457,11 @@
     "copy-from": "smg_base",
     "looks_like": "hk_mp5",
     "type": "GUN",
-    "name": { "str": "MP 40" },
-    "description": "This is a 9x19mm submachine gun with a folding stock, which was used extensively during Second World War.",
+    "name": { "str": "Schmeisser MP 40" },
+    "description": "This is a simple and renowned 9x19mm submachine gun developed by Hugo Schmeisser for the German army.  Adopted  in 1940, it saw extensive service on both fronts of the Second World War and was much regarded by soldiers who fielded the weapon in combat, being praised for its reliability and relatively modern design.  Featuring an integral folding wire butt stock, this weapon more than likely entered the United States as a war trophy by a returning veteran.",
     "weight": "3969 g",
     "volume": "2320 ml",
-    "longest_side": "648 mm",
+    "longest_side": "348 mm",
```

```suggestion
    "longest_side": "648 mm",
```
FOLDED_STOCK or COLLAPSED_STOCK handle the shrinkage when folded